src/Helpers/Portfolio/PortfolioHelper.js

```diff
@@ -59,3 +59,16 @@ export async function updatePortfolio(portfolioData) {
 export async function removePortfolio(portfolioId) {
   await userApi.destroyPortfolio(portfolioId);
 }
+
+export async function removePortfolioItem(portfolioItemId) {
+  return userApi.destroyPortfolioItem(portfolioItemId).then(data => {
+    console.log('Remove Portfolio Item Called successfully.');
```

Comment:
console log :wink: 

And after you remove it you don't need the `.then` code. Just return the `userApi.destroyPortfolioItem(portfolioItemId)` API call.